ui/src/react-admin/modules/navigation/components/NavigationEditForm/NavigationEditForm.tsx

```diff
@@ -137,6 +147,15 @@ const NavigationEditForm: FunctionComponent<NavigationEditFormProps> = ({
 					}}
 				/>
 			</FormGroup>
+			<FormGroup label={tText('Taal')} error={formErrors.language}>
+				<Select
```

Comment:
Judging by the screenshots, this `Select` is unstyled, I'm anticipating that'll be a remark during testing but I'm unsure of exact scope here.

Reply:
It should be unstyled in the admin-core
We only provide minimal styling in the admin-core to make the component functional
the rest of the styling will be done in the respective clients: avo-client or hetarchief-client

This is so we don't have to focfully override styles from the admin-core to match the look and feel of the respective client